src/index.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  Logger,
+  Payload,
+  UnexpectedError,
+  Workflow,
+  makePayload,
+  makeWorkflow
+} from "./workflow";
+
+export type Logger = Logger;
+export type Payload<Input> = Payload<Input>;
+export type UnexpectedError = UnexpectedError;
+export type Workflow<Input, Result> = Workflow<Input, Result>;
+
+module.exports = {
```

Comment:
this has been done this way to avoid redondant type annotation in this file.

Reply:
I dont' understand the link between redondant type annotation and the use of `module.exports`? 🤔

Reply:
when I tried to use an `export` statement the compiler requested to add more type annotations.

Reply:
And you saw it done like that in other places? I'm curious about the errors, I must admit.